youtube_dl/extractor/bilibili.py

```diff
@@ -46,58 +30,42 @@ class BiliBiliIE(InfoExtractor):
             'title': '【金坷垃】金泡沫',
             'description': 'md5:ce18c2a2d2193f0df2917d270f2e5923',
             'duration': 308.067,
-            'timestamp': 1398012678,
+            'timestamp': 1397983878,
```

Comment:
Why is there such a value here? Shouldn't it be automated?

Reply:
this is the test case, when I ran the test, it complained about time stamp not match , and when i check the video info, found the video timestamp has change since last commit.